util/src/main/java/io/kubernetes/client/util/Yaml.java

```diff
@@ -59,6 +58,100 @@ public class Yaml {
 
   static final Logger logger = LoggerFactory.getLogger(Yaml.class);
 
+  /**
+   * Load an API object from a YAML string representation without already knowing the object type and loads and submit any type of resource
+   * V1Pod
+   *
+   * @param content The YAML content
+   * @return An instantiation of the object.
+   * @throws IOException If an error occurs while reading the YAML.
+   */
+  public static Object loadAndSubmitResource(String content) throws Exception {
+    //loading yaml content as unstructured object
+    Object unstructuredObject = Yaml.load(new StringReader(content));
+
+    if (!(unstructuredObject instanceof Map)) {
```

Comment:
fwiw, this will only work for single-instance YAML files, if people put multiple objects in a single file, it will fail. That's fine for a starting point, but you may want to document it somewhere.